test/unit/layout/toggleHeader.spec.js

```diff
@@ -0,0 +1,36 @@
+
```

Comment:
Can we also edit the test's filenames too?

Reply:
done

Reply:
You haven't changed the casing of any of the filenames...?